app/webpacker/components/wca/UtcDatePicker.js

```diff
@@ -0,0 +1,74 @@
+import React, { useCallback, useMemo } from 'react';
+
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+import { DateTime } from 'luxon';
+
+const loadAsPseudoLocal = (isoString) => {
+  if (!isoString) { return false; }
+
+  // all of our WCIF-contained dates are defined to be UTC
+  // eslint-disable-next-line implicit-arrow-linebreak
+  return DateTime.fromISO(isoString, { zone: 'UTC' })
+    // but the react-datepicker uses local TZ,
+    // so we have to make the date _think_ it's local without actually converting the time
+    .setZone('local', { keepLocalTime: true })
+    // and finally output as JS-compatible object
+    .toJSDate();
+};
+
+const useIsoDate = (isoString) => useMemo(() => loadAsPseudoLocal(isoString), [isoString]);
```

Comment:
Why can't we just directly use `loadAsPseudoLocal` everywhere? The performance won't be affected by recalculating, and there aren't any side effects to prevent.

Reply:
The intention was to make sure that we don't re-calculate all four dates if only one date changes. But indeed I might have overlooked something, because my experience with render side effects in React is rather low.

Reply:
I'll merge this for now and leave room for future optimization.